samples/batch/transcription-enabled-storage/CreateByTimerFunction/CreateTranscriptionByTimer.cs

```diff
@@ -0,0 +1,348 @@
+
+                var errorTxtName = jobName + "_error.txt";
+                var responseMessage = $"Transcription failed with FormatException:\n{e.Message}";
+
+                var errorOutputContainer = CreateTranscriptionEnvironmentVariables.ErrorReportOutputContainer;
+                await StorageUtilities.WriteTextFileToBlobAsync(CreateTranscriptionEnvironmentVariables.AzureWebJobsStorage, responseMessage, errorOutputContainer, errorTxtName, log).ConfigureAwait(false);
+
+                return string.Empty;
+            }
+            catch (TimeoutException e)
+            {
+                log.LogError($"Timeout while creating post, re-enqueueing transcription create. Message: {e.Message}");
+                var createTranscriptionSBConnectionString = CreateTranscriptionEnvironmentVariables.CreateTranscriptionServiceBusConnectionString;
+
+                foreach (var message in messages)
+                {
+                    ServiceBusUtilities.SendServiceBusMessageAsync(createTranscriptionSBConnectionString, message, log, 2).GetAwaiter().GetResult();
+                }
+
+                return string.Empty;
+            }
+
+            var reenqueueingTimeInSeconds = MapDatasetSizeToReenqueueingTimeInSeconds(CreateTranscriptionEnvironmentVariables.AudioDatasetSize);
+
+            var transcriptionMessage = new PostTranscriptionServiceBusMessage(
+                subscription,
+                locationString,
+                jobName,
+                StartDateTime.ToString(CultureInfo.InvariantCulture),
+                locale,
+                modelIds.Any(),
+                CreateTranscriptionEnvironmentVariables.AddSentimentAnalysis,
+                CreateTranscriptionEnvironmentVariables.AddEntityRedaction,
+                0,
+                reenqueueingTimeInSeconds,
+                reenqueueingTimeInSeconds);
+            var fetchTranscriptionSBConnectionString = CreateTranscriptionEnvironmentVariables.FetchTranscriptionServiceBusConnectionString;
+            log.LogInformation($"FetchTranscriptionServiceBusConnectionString from settings: {fetchTranscriptionSBConnectionString}");
+
+            ServiceBusUtilities.SendServiceBusMessageAsync(fetchTranscriptionSBConnectionString, transcriptionMessage.CreateMessageString(), log, reenqueueingTimeInSeconds).GetAwaiter().GetResult();
+            log.LogInformation($"Fetch transcription queue informed about job at: {jobName}");
+
+            return locationString;
+        }
+
+        internal static bool IsValidServiceBusMessage(Message message)
+        {
+            var messageBody = Encoding.UTF8.GetString(message.Body);
+
+            try
+            {
+                var serviceBusMessage = JsonConvert.DeserializeObject<ServiceBusMessage>(messageBody);
+
+                if (serviceBusMessage.EventType.Contains("BlobCreate", StringComparison.OrdinalIgnoreCase) &&
```

Comment:
So currently we do not handle that file content has been changed. What is about renaming?

Reply:
I don't understand. Can you elaborate?

Reply:
closing as discussed